config/printer-sovol-sv06-2022.cfg

```diff
@@ -0,0 +1,172 @@
+# This file contains pin mappings for the stock Sovol SV06
+# To use this config, during "make menuconfig" select the 
+# STM32F103 with a "28KiB bootloader" and serial (on USART1 PA10/PA9) communication.
+# Also, since it is using the GD32F103, please select Disable SWD at startup
+
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
+# turning on the printer with the card inserted. The firmware
+# filename must end in ".bin" and must not match the last filename
+# that was flashed.
+
+# See docs/Config_Reference.md for a description of parameters.
+[include mainsail.cfg]
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0
+restart_method: command
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 3000
+max_z_velocity: 5
+max_z_accel: 100
+
+[stepper_x]
+step_pin: PC2
+dir_pin: !PB9
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: tmc2209_stepper_x:virtual_endstop
+position_endstop: 0
+position_max: 225
+homing_speed: 40
+homing_retract_dist: 0
+
+[tmc2209 stepper_x]
+uart_pin: PC1
+run_current: 0.860
+stealthchop_threshold: 180
+interpolate: False
+sense_resistor: 0.150
+uart_address: 3
+driver_SGTHRS: 81
+diag_pin: PA5
+
+[stepper_y]
+step_pin: PB8
+dir_pin: PB7
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: tmc2209_stepper_y:virtual_endstop
+position_endstop: 0
+position_max: 225
+homing_speed: 40
+homing_retract_dist: 0
+
+[tmc2209 stepper_y]
+uart_pin: PC0
+run_current: 0.900
+stealthchop_threshold: 180
+interpolate: False
+sense_resistor: 0.150
+uart_address: 3
+driver_SGTHRS: 82
+diag_pin: PA6
+
+[stepper_z]
+step_pin: PB6
+dir_pin: !PB5
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 4
+endstop_pin: probe:z_virtual_endstop
+position_min: -4
+position_max: 261
+homing_speed: 4
+
+[tmc2209 stepper_z]
+uart_pin: PA15
+run_current: 1.000
+stealthchop_threshold: 40
+interpolate: False
+sense_resistor: 0.150
+uart_address: 3
+diag_pin: PA7
+
+[thermistor sovol_thermistor]
+temperature1: 25
+resistance1: 94162
+beta: 4160
+
+[extruder]
+max_extrude_only_distance: 100.0
+step_pin: PB4
+dir_pin: !PB3
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 4.56
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PA1
+sensor_type: sovol_thermistor
```

Comment:
Change sensor_type to whatever sensor this printer uses, based off the unneeded thermistor section, its probably a generic EPCOS 100K B57560G104F

Reply:
I just defined it the same way Sovol did in their Marlin source.

Reply:
Marlin does its own thing, for klipper you can just change sensor_type to that epcos thermistor I said and it will simplify things while working the same

Reply:
sensor type updated